scripts/search.js

```diff
@@ -6,13 +6,13 @@
     var $searchInput = $($selector);
 
     // config is https://github.com/algolia/docsearch-configs/blob/master/configs/babeljs.json
+    var _0xac38 = ["\x64\x34\x32\x39\x30\x36\x62\x30\x34\x33\x63\x35\x34\x32\x32\x65\x61\x30\x37\x62\x34\x34\x66\x64\x34\x39\x63\x34\x30\x61\x30\x64", "\x62\x61\x62\x65\x6C\x6A\x73"];
     var $search = docsearch({
-        apiKey: 'd42906b043c5422ea07b44fd49c40a0d',
```

Comment:
Theses keys are meant to be public.

I'm not a fan of this because it will be more complicated to change the keys.